ConferenceV2/app/conf_logger.py

```
@@ -42,16 +42,14 @@ def __init__(self, version="1"):
             return False
 
-        # Azure App Insights handler
         try:
-            azure_handler = AzureLogHandler(connection_string=connection_string)
-            azure_handler.setLevel(logging.DEBUG)  # Set log level for production
-            self.logger.addHandler(azure_handler)
+            if not any(isinstance(h, AppInsightsLogHandler) for h in self.logger.handlers):
+                self.logger.addHandler(AppInsightsLogHandler())
             return True
-        except (ValueError, TypeError):
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)